server/tests/unit/services/RunService.test.js

```diff
@@ -236,6 +237,85 @@ describe('RunService', () => {
                 expect(tech.active).toBe(true);
             });
         });
+        it('creates several runs from a list of apg examples', async () => {
+            await dbCleaner(async () => {
+            
+                const browserVersionNumber = '1.2.3';
+                const atVersionNumber = '3.2.1';
+
+                let testVersion = await db.TestVersion.findOne({
+                    where: {
+                        git_hash: process.env.IMPORT_ARIA_AT_TESTS_COMMIT_1
+                    }
+                });
+                const apgExamples = await db.ApgExample.findAll({
+                    where: { test_version_id: testVersion.id }
+                });
+
+                const apgExampleIds = apgExamples.map(example => example.dataValues.id)
+
+                const at = await db.At.findOne({
+                    where: { test_version_id: testVersion.id },
+                    include: [db.AtName]
+                });
+
+                const browser = await db.Browser.findOne();
+
+                const runStatus = await db.RunStatus.findOne({
+                    where: { name: db.RunStatus.RAW }
+                });
+
+                const activeRuns = await RunService.configureRuns({
+                    test_version_id: testVersion.id,
+                    apg_example_ids: apgExampleIds,
+                    at_browser_pairs: [
+                        {
+                            at_name_id: at.at_name_id,
+                            at_version: atVersionNumber,
+                            browser_id: browser.id,
+                            browser_version: browserVersionNumber
+                        }
+                    ]
+                });
+
+                // There are three apg examples
+                const keys = Object.keys(activeRuns);
+                expect(keys.length).toEqual(3);
+
+                // TODO: is there a better way to assert this?
+                Object.values(activeRuns).forEach((run, i) => {
```

Comment:
I know it is super verbose, but its best not to be using higher-order functions like `forEach` in tests. 

The code is the correct place for this kind of abstraction. But not the tests itself. 

https://mtlynch.io/good-developers-bad-tests/#dare-to-violate-dry

Also if you want to ignore the Id instead of deleting the id you could use `objectContaining`: https://jestjs.io/docs/en/expect#expectobjectcontainingobject